code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -326,24 +326,47 @@
 			if((A == src) || istype(A, /mob))
 				continue
 			return
-	move_outside(usr, T)
+	if(occupant == usr)
+		move_outside(usr, T)
+	else if(passengers.Find(usr))
+		move_passenger_outside(usr, T)
 
 /obj/spacepod/proc/move_outside(mob/living/user, turf/exit_loc = src.loc)
 	if(occupant)
 		inertia_dir = 0 // engage reverse thruster and power down pod
 		occupant.forceMove(exit_loc)
 		occupant = null
 		to_chat(usr, "<span class='notice'>You climb out of the pod.</span>")
+		if(passengers.len)
+			for(var/mob/living/L in passengers.len)
+				if(!occupant)
+
+					var/answer = alert(L,"The pilot has evacuated \the [src]. Do you wish to take over?",,"Yes","No")
```

Comment:
Yes, use action buttons
But if you want an example of how to conditionally add verbs, check the spacepod equipment file
I don't remember how it's done but I know it's done in there